examples/socket-mode-oauth/app.js

```diff
@@ -0,0 +1,32 @@
+const { App, SocketModeReceiver } = require('@slack/bolt');
+
+const socketModeReceiver = new SocketModeReceiver({
+  appToken: process.env.SLACK_APP_TOKEN,
+  clientId: process.env.SLACK_CLIENT_ID,
+  clientSecret: process.env.SLACK_CLIENT_SECRET,
+  stateSecret: 'my-state-secret',
+  scopes: ['channels:history', 'chat:write', 'commands'],
+  customRoutes: [
+    {
+      path: '/health-check',
+      method: ['GET'],
+      handler: (req, res) => {
+        res.writeHead(200);
+        res.end('Health check information displayed here!');
+      },
```

Comment:
A meaningful health check logic can be like this.

However, `client.badConnection` is a private property. Before recommending something like this, we need to enhance `@slack/socket-mode` to have a proper property/method for this purpose. I prefer adding `public isActive(): boolean` rather than exposing `badConnection: boolean`.

```suggestion
      handler: (req, res) => {
        if (socketModeReceiver.client.badConnection) {
          res.writeHead(503);
          res.end('The Socket Mode client is not connected.');
        } else {
          res.writeHead(200);
          res.end('OK');
        }
      },
```

Reply:
`customRoutes` added back to the `App` level